src/convertToObject.js

```diff
@@ -6,7 +6,21 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const declarations = sourceString.split(';');
+  const styleObject = {};
+
+  for (const declaration of declarations) {
+    const parts = declaration.split(':');
+
+    const key = parts[0].trim();
+    const value = parts[1] ? parts[1].trim() : '';
+
+    if (key && value) {
+      styleObject[key] = value;
+    }
```

Comment:
You can simplify your code like this
```suggestion
    const [key, value] = declaration.split(':').map((el) => el.trim());

    if (key && value) {
      styleObject[key] = value;
    }
```

Reply:
thank you